server/src/GraphQL/Mutations/Checkout.php

```diff
@@ -45,12 +46,26 @@ public function checkout($rootValue, array $args, GraphQLContext $context, Resol
                 ]
             );
 
-            // TODO: Should we pass all the data on at this point?
+            // Get all the data that was modified by the event service and corresponding listeners
             $eventServiceData = $eventService->getAllModifiedData();
 
             $createCheckout = App::make(CreateCheckout::class);
             $order = $createCheckout->handle($args['input']);
 
+            // Attach the transaction data to the order_transactions table
```

Comment:
@AlexeiaW I think we'll need a string here to represent which payment provider was used for the order and to insert that into the transaction table. The string would have to be unique, something like "stripe_standard" or whatever makes sense in the context.